### PR TITLE
Fix broken links that use PlatformVers attribute in 2.4 docs

### PR DESCRIPTION
**Pull request by ariordan-redhat**
Fix the links in 2.4 that use the `{PlayformVers}` attribute

### CODE REVIEW

**Comment by ariordan-redhat** [comment]
List of links to fix in this PR:
- [ X] ./modules/dev-guide/con-content-workflows.adoc:https://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform/{PlatformVers}/html/ansible_navigator_creator_guide
  - Changed to html/ansible_navigator_creator_guide/index
- [X] ./modules/platform/ref-hub-configs.adoc:https://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform/2.4/html/managing_red_hat_certified_and_ansible_galaxy_collections_in_automation_hub/index
  - Changed to Managing Red Hat Certified, validated, and Ansible Galaxy content

- [X] ./modules/platform/con-aap-upgrade-prereq.adoc:https://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform/2.4/html/red_hat_ansible_automation_platform_installation_guide/planning-installation#proc-attaching-subscriptions_planning
  - Updated

- [ ] ./modules/platform/proc-editing-inventory-file-for-updates.adoc:https://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform/2.4/html/red_hat_ansible_automation_platform_installation_guide/planning-installation#supported_installation_scenarios
- [ ] ./modules/platform/proc-installing-the-aap-setup-bundle.adoc:https://access.redhat.com/downloads/content/480
- [ ] ./modules/platform/proc-installing-without-internet.adoc:https://access.redhat.com/downloads/content/480
- [ ] ./modules/platform/proc-upgrade-installer.adoc:https://access.redhat.com/downloads/content/480
- [ ] ./modules/platform/proc-choosing-obtaining-installer.adoc:https://access.redhat.com/downloads/content/480
- [ ] ./modules/platform/proc-choosing-obtaining-installer.adoc:https://access.redhat.com/downloads/content/480
- [ ] ./modules/platform/proc-installing-with-internet.adoc:https://access.redhat.com/downloads/content/480
- [ ] ./assemblies/platform/assembly-migrate-legacy-venv-to-ee.adoc:https://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform/2.4/html/ansible_navigator_creator_guide
- [ ] ./assemblies/platform/assembly-supported-installation-scenarios.adoc:https://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform/2.4/html/red_hat_ansible_automation_platform_installation_guide/assembly-multi-machine-cluster-scenario
- [ ] ./assemblies/platform/assembly-setting-up-automation-mesh.adoc:https://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform/2.4/html/red_hat_ansible_automation_platform_installation_guide/planning-installation#red_hat_ansible_automation_platform_system_requirements
- [ ] ./assemblies/platform/assembly-supported-installation-scenarios.adoc:https://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform/2.4/html/red_hat_ansible_automation_platform_installation_guide/single-machine-scenario#assembly-standalone-controller-ext-database
- [ ] ./assemblies/platform/assembly-supported-installation-scenarios.adoc:https://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform/2.4/html/red_hat_ansible_automation_platform_installation_guide/single-machine-scenario#assembly-standalone-hub-ext-database
- [ ] ./assemblies/platform/assembly-supported-installation-scenarios.adoc:https://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform/2.4/html/red_hat_ansible_automation_platform_installation_guide/single-machine-scenario#standalone-controller-non-inst-database
- [ ] ./assemblies/hub/assembly-validated-content.adoc:https://access.redhat.com/downloads/content/480
- [ ] ./titles/hub/install/master.adoc:https://access.redhat.com/downloads/content/480 (redirect is set up)
  - Change to the destination set up in the redirect

File: downstream/modules/platform/ref-hub-configs.adoc [hunk 1]
```diff
@@ -9,6 +9,6 @@ See the following resources to explore additional {HubName} configurations.
 |====
 |Resource link|Description
 |link:https://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform/{PlatformVers}/html/managing_user_access_in_private_automation_hub/index[Managing user access in private {HubName}]|Configure user access for {HubName}
-|link:https://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform/{PlatformVers}/html/managing_red_hat_certified_and_ansible_galaxy_collections_in_automation_hub/index[Managing Red Hat Certified and Ansible Galaxy collections in {HubName}]|Add content to your {HubName}
+|link:https://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform/{PlatformVers}/html/managing_red_hat_certified_validated_and_ansible_galaxy_content_in_automation_hub/index/index[Managing Red Hat Certified, validated, and Ansible Galaxy content in {HubName}]|Add content to your {HubName}
```
Comment (dcdacosta) [inline downstream/modules/platform/ref-hub-configs.adoc:12]: Remove the extra /index here which should fix the link.

**Review by dcdacosta — CHANGES_REQUESTED** [review]
@ariordan-redhat One minor correction needed and these are Ok. Since this needs a change, if you want to include other fixes here, I can review them when ready.

**Comment by ariordan-redhat** [comment]
@dcdacosta

Corrections are done. Can you check the newer links, please?
I didn't modify some outdated links that were commented out in `assemblies/platform/assembly-supported-installation-scenarios.adoc`

**Review by dcdacosta — APPROVED** [review]
Affected books built and link fixes verified. LGTM!

**Pull request merged**